src/cfnlint/formatters/__init__.py

```diff
@@ -2,11 +2,29 @@
 Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 SPDX-License-Identifier: MIT-0
 """
+import itertools
 import json
+import operator
+import sys
 from junit_xml import TestSuite, TestCase, to_xml_report_string
 from cfnlint.rules import Match
 
 
+class color(object):
+    error = '\033[31m'
+    warning = '\033[33m'
+    informational = '\033[34m'
+    unknown = '\033[37m'
+    green = '\033[32m'
+    reset = '\033[0m'
+    bold_green = '\033[1;32m'
```

Comment:
Couldn't find much shell color documentation more authoritative than https://gist.github.com/vratiu/9780109

Any other resource you used?

Reply:
Not really.  https://github.com/tartley/colorama/blob/master/colorama/ansi.py#L49-L58 looked at other color packages too.